### ISSUE TITLE
"Tabbed interface" tab content disappears when Open/Close all function is used

### ISSUE THREAD

**Issue by josephbr** [wet-boew/wet-boew#6542]
I seem to have found a strange problem when tabs (using the "Tabbed Interface" code) have been nested inside of collapsible bars and the "CLOSE ALL" function is invoked.

The OPEN/CLOSE ALL function I am using is also referenced in this ticket:
https://github.com/wet-boew/wet-boew/issues/6416

This is the button's markup is:
```
<button type="button" class="wb-toggle btn-link" data-toggle='{"selector": "details", "parent": ".parent-boxTabs1", "type": "off", "print": "on"}'>CLOSE ALL</button>
```

Upon clicking it, the collapsible bars close (as they should) but the tabs are affected as well. In other words, I manually open the bar after clicking "CLOSE ALL", the tabs' content will appear blank. If I click "OPEN ALL", then it re-appears suggesting that the tabs are somehow being collapsed as well.

**Comment by hsrudnicki** [comment]
Can this issue be closed?

**Comment by pjackson28** [comment]
@josephbr Has this issue been fixed now that #6416 has been fixed? Can you still reproduce the issue with the latest version?

**Comment by josephbr** [comment]
The design ended up changing to NOT use nested tabs in collapsible bars so I am not sure if this has been fixed.

**Comment by jfbourgon** [comment]
Can I get an update on the status of this issue?

I'm asking because I'm trying to embed a toggle button inside a tabbed interface and I'm getting awkward behavior. When the toggle button is toggle of it's propagating off state to tab surrounding it and that makes the content of other tab to disappear. Here is the code that could allow you to replicate the issue:

```html
<div class="wb-tabs ignore-session">
    <div class="tabpanels">
        <details id="tab-search">
        <summary>Tab #1</summary>
            <div class="container-fluid">
                <p>Tab 1 content</p>
            </div>  
        </details>
        <details id="tab-results">
        <summary>Tab #2</summary>
            <div class="container-fluid">
                <p>Tab 2 content</p>
            </div>
        </details>
        <details id="tab-edit">
            <summary>Tab #3</summary>
            <div class="container-fluid">
                <button type="button" class="btn btn-primary btn-sm wb-toggle" data-toggle='{ "selector": "p", "parent": "#toggle-container", "stateOn": "show", "stateOff": "hidden" }'>Toggle</button>
                <div id="toggle-container">
                    <p>Hello!</p>
                </div>
            </div>
        </details>
    </div>
</div>
<script>
$( document ).on( "wb-ready.wb", function( event ) {   
    $( "#toggle-container p" ).addClass( "hidden" ); 
});
</script>
```

When the toggle button is toggled off the `class=off` is propagated on inactive tab details and their content then become invisible so the appear as empty when we switch back to them. Toggling on the toggle button doesn't bring back the content. According to the documentation the state switch should be limited to the parent and selector defined in the plugin configuration.

1) Initial state
![image](https://cloud.githubusercontent.com/assets/1030226/9588325/9c5e112c-4ff5-11e5-976a-9f014c7a5ed5.png)

2) Tab 3 containing a toggle button
![image](https://cloud.githubusercontent.com/assets/1030226/9588380/ed90bd2e-4ff5-11e5-93cf-b18653a8b310.png)

3) Empty Tab 1 and Tab 2 once the off state is triggered
![image](https://cloud.githubusercontent.com/assets/1030226/9588407/15633232-4ff6-11e5-8643-4c98b8d37765.png)

**Comment by jfbourgon** [comment]
Would it be possible to get any update on this issue. I initially thought that the observed behavior was related to the same issue but this might not the case. Any advice on possible workaround would be highly appreciated.

I also confirm that WET 4.0.17 is not solving the issue.

**Comment by pjackson28** [comment]
Remaining issue will be fixed by PR #7150.
